staging/src/github.com/kubeedge/viaduct/pkg/conn/ws.go

```diff
@@ -114,8 +114,8 @@ func (conn *WSConnection) handleRawData() {
 }
 
 func (conn *WSConnection) handleMessage() {
-	msg := &model.Message{}
```

Comment:
Is it necessary?

Reply:
yes, it is a bug  for handling message. The bug is that each time a message is received, the information of the last message will be retained